coralproject/talk
# save dismissed timer note to localStorage

## What does this PR do?

When you click the timer bar on the dashboard, it stays dismissed. I saved it to localStorage.

## How do I test this PR?

- go to the Dashboard. click the X to dismiss
- reload. it should still be gone
- do the same thing in Safari Private mode. it shouldn't break.
